### ISSUE TITLE
Impossible to interrupt process of building, when used BuildKit

### ISSUE THREAD

**Issue by saruman9** [loft-sh/devspace#1387]
**What happened?**

Impossible to interrupt process of building (with Ctrl-C) when used BuildKit.

**What did you expect to happen instead?**

Process of building should be interrupted.

**How can we reproduce the bug?**

1. Use `useBuildKit` option for local building Docker engine.
2. Run a process of building.
3. Try to interrupt this by pressing Ctrl-C.

**Local Environment:**  
- DevSpace Version: 5.10.0
- Operating System: linux
- Deployment method: helm | kubectl apply

**Kubernetes Cluster:**
- Kubernetes Version: 
```
Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.5", GitCommit:"6b1d87acf3c8253c123756b9e61dac642678305f", GitTreeState:"archive", BuildDate:"2021-03-25T19:02:48Z", GoVersion:"go1.16.2", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.2", GitCommit:"faecb196815e248d3ecfb03c680a4507229c2a56", GitTreeState:"clean", BuildDate:"2021-01-13T13:20:00Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"linux/amd64"}
```

/kind bug

**Comment by FabianKramm** [comment]
@saruman9 thanks for reporting this issue! I can reproduce it on linux, but it will need more investigation to find out the root cause of this. As a workaround you could use `images.build.buildKit` instead which works fine on linux

**Comment by saruman9** [comment]
Yes, I switched to BuildKit engine and have not the problem yet.

**Comment by deniseschannon** [comment]
Closing due to inactivity. If this is still an issue, please open a new issue.
